templates/VALIDATION.md

```diff
@@ -100,7 +102,7 @@ kind: Template
         [
           {
             “rule”: “integer”,
-            "path": "/VM/domain/cpu/cores",
+            "path": "/spec.domain.cpu.cores",
```

Comment:
If the root is the objects: node then I think this won't work, because the objects: node is an array. It might work without the / though